safe/impact_function/impact_function.py

```diff
@@ -1442,16 +1444,17 @@ def aggregation_preparation(self):
                         if count_key not in keywords['inasafe_fields']:
                             # The exposure hasn't a count field, we should add
                             # it.
-                            default = definition(ratio_field)['default_value']
+                            default_value = get_inasafe_default_value_qsetting(
```

Comment:
Just trying to remember, so even if there is no ratio (e.g. no youth_ratio), it will take from the setting?